src/routes/+page.svelte

```diff
@@ -51,22 +63,43 @@
 			<Button label="Read more on Blog" href="/blog" slot="footer" />
 		</Section>
 	{/if}
+
+	<Section class="newsletter" id="home-newsletter" layout="boxed" padding="md">
+		<NewsletterOptInBanner />
+	</Section>
 </main>
 
 <style lang="postcss">
+	:global(html) {
+		scroll-behavior: smooth;
+	}
```

Comment:
FYI – this still "leaks" the `scroll-behavior` onto the rest of the app. Expect any un-scoped `global` CSS in any component to affect your entire app.

Since `scroll-behavior` must be attached to the root element, scoping it is it bit tricker. Here's an option that works well – keeps the relevant CSS co-located in the same component, prevents "leakage":

```css
	:global html:has(#home-newsletter) {
		scroll-behavior: smooth;
	}
```

No need to change this – I have it in a local commit and will push shortly.

Reply:
okay! although `:has` tag is only supported by Safari/WebKit...

Or does PostCSS Preset Env make this work??

Reply:
> Or does PostCSS Preset Env make this work??

indeed 😄

Anything stage 2 or above here should work:
https://cssdb.org/

That said… I've seen some inconsistent/buggy behavior with `:has` pseudo – so use with caution and test it!